maven-extension-rollout/rollout.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+commit_msg="Apply Gradle Enterprise Maven Extension"
+basedir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
I believe `BASH_SOURCE[0]` is not available with bash older than v3. So depending on where the script runs you could replace it with `$0`. OTOH, `BASH_SOURCE[0]` is "better" than `$0`.